_layouts/project.html

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+---
+<article class="post">
+  <header class="post-header">
+    <h1 class="project-title">{{ page.title | escape}} </h1>
+    <h2 class="project-creator-names">{{ page.creator_names | escape}}</h2>
```

Comment:
Since we sometimes have more than one creator, this doesn't quite look right. Check out how the projects with multiple creators look now. Maybe something like "Name1, Name2, and Name3"?